eth/v1alpha1/beacon_chain.proto

```diff
@@ -288,7 +288,8 @@ message ListValidatorAssignmentsRequest {
 
 message ValidatorAssignments {
     message CommitteeAssignment {
-        // 
+        // The list of validator indices that are in the same committee as requested validator,
+        // everyone in the committee is assigned to attest at the same slot and same shard.
```

Comment:
Can you add what a cross link committee is? Is this a group of validators that need to produce a crosslink? It's not clear from this context.